### PR TITLE
AM-2926 verbose error messages vol 2

### PR DESCRIPTION
**Pull request by lgw-gravitee**
https://gravitee.atlassian.net/browse/AM-2926

Tested request:

```
PATCH http://{AM_HOST}/management/organizations/DEFAULT/environments/DEFAULT/domains/{DOMAIN_ID}
{"webAuthnSettings":{"certificates":{}"origin":"http://localhost:8092","relyingPartyId":"localhost","forceRegistration":true}}
```
Request is malformed - missing “,” after “certificates”:{}

The result:

```
Unexpected character ('"' (code 34)): was expecting comma to separate Object entries
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 40] (through reference chain: io.gravitee.am.service.model.PatchDomain["webAuthnSettings"])
```

Expected result:
{"message":"Malformed request body","http_status":400}

### CODE REVIEW

**Review by leleueri — COMMENTED** [review]
suggestion: Looks like the 3 exceptions are inheriting from the same JacksonException.
Maybe we can do a single mapper using this super class ?

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/provider/JsonProcessingExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import io.gravitee.common.http.HttpStatusCode;
+
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.Provider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Titouan COMPIEGNE (titouan.compiegne at graviteesource.com)
```
Comment (podlesrafal) [inline gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/provider/JsonProcessingExceptionMapper.java:30]: change author

Comment (lgw-gravitee) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/provider/JsonProcessingExceptionMapper.java:30]: Its not mine:P Git handled it in wrong way :P

Comment (lgw-gravitee) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/provider/JsonProcessingExceptionMapper.java:30]: I guess GIT handled it in wrong way. Im not creator of that class

**Comment by lgw-gravitee** [comment]
> suggestion: Looks like the 3 exceptions are inheriting from the same JacksonException. Maybe we can do a single mapper using this super class ?

The issue is that there are other JsonParseExceptionMapper and JsonMappingExceptionMapper that take precedence over the general JacksonExceptionMapper. They need to be overridden.

**Comment by lgw-gravitee** [comment]
I manged to configure app without adding default exception mappers. Now its working with general exception mapper

**Review by leleueri — APPROVED** [review]

**Pull request merged**